packages/react-composites/tests/browser/chat/app/identity.ts

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
Are we keeping these headers or no? I recall this was a bit of a conversation earlier.

Reply:
I get eslint errors if we dont.